p/themes/base-theme/template.css

```diff
@@ -172,7 +172,10 @@ button.as-link[disabled] {
 
 /*=== Tables */
 table {
+	display: block;
```

Comment:
A table with display: block will not behave correctly because you're telling it not to behave like a table. It doesn't look like you'll be able to do this without a container element.

table-layout: fixed might also help, potentially. Give it a try or remind me another day. ;-)

Reply:
I did some tests and there doesn't seem to be a way to avoid a container element to see fully correct behavior. If there were a decent method I overlooked I imagine they'd be using it over at https://getbootstrap.com/docs/5.0/content/tables/#always-responsive for example.

We could regard the proposal in this PR as a necessary evil since the internals of the table seem to behave properly, at least at first glance, even with colspan and rowspan. That means the crime's hopefully hidden as long as you ensure background and border on the `<table>` element itself are transparent.

![Screenshot_2021-09-02_10-01-19](https://user-images.githubusercontent.com/202757/131805954-9761d08a-3938-4c3e-96f2-e3905aabdbdc.png)

I'd probably feel better simply using a container element, but then for feed content question becomes whether playing around with `cloneNode()` for example is a bigger evil. It wouldn't be much effort in any case. But the proposal from #3129  also sounds decent enough and very clean, or are there any issues with that?

For reference, here's a simple table.
```html
<table>
	<thead>
		<tr>
			<th colspan="2">Table header</th>
		</tr>
	</thead>
	<tbody>
		<tr>
			<td rowspan="2">Table body</td>
			<td>Second column</td>
		</tr>
		<tr>
			<td>Second column</td>
		</tr>
	</tbody>
</table>
```

Reply:
on which browser `display:block` does not work?

source of my solution: https://stackoverflow.com/questions/5533636/add-horizontal-scrollbar-to-html-table

Reply:
That page even lists additional problems I wasn't aware of, most notably https://www.tpgi.com/short-note-on-what-css-display-properties-do-to-table-semantics/ 

Here are a few quick examples showing some ways in which display: block does not behave how a table should behave.

[table.zip](https://github.com/FreshRSS/FreshRSS/files/7100531/table.zip)

There are more of course. You can stick a width on the surrounding block all you want, the table contents won't grow or shrink properly.

Reply:
OK, I will revert to 8412836

But it will not fix #3129

For the user list (see https://github.com/FreshRSS/FreshRSS/pull/3819#issuecomment-910741397) I will try to find another solution.

Reply:
Simply a container around it will do the trick.